qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -208,6 +208,13 @@
                         generic_blacklist += 'generic/070\ngeneric/650\n'
                     with_cache.always..with_nfs_source.multi_fs:
                         generic_blacklist += 'generic/451\n'
+                    # These tests not support mmap with cache=none
+                    with_cache.none..with_nfs_source.multi_fs:
```

Comment:
Should this condition be independent of `with_nfs_source.multi_fs`?
If there is no correlation, why don't we use the filter below?
`with_xfstest..with_cache.none:`

Reply:
Yes, the `with_xfstest..with_cache.none` filter is more suitable, and I think the above filter (line 209) has the same problem，can I fix these two problems in this commit?

Reply:
If they are for the same reason, I think they can be merged together.